lib/DDG/Spice/Symbolab.pm

```diff
@@ -128,6 +165,15 @@ handle query_lc => sub {
 	else{
 		return ;
 	}
+}
+
+spice is_cached => 1;
+spice to => 'https://www.symbolab.com/ddg?query=$1';
+spice wrap_jsonp_callback => 1;
+
+triggers startend => keys %legalWords, keys %legalFunctions;
+handle query_lc => sub {
+    return isLegal($_);
```

Comment:
nit: seems like we might as well get rid of the `isLegal` fn and just put the logic in the `handler`